pkgs/applications/networking/browsers/firefox/default.nix

```diff
@@ -64,7 +65,6 @@ stdenv.mkDerivation rec {
       "--disable-necko-wifi" # maybe we want to enable this at some point
       "--disable-installer"
       "--disable-updater"
-      "--disable-pulseaudio"
```

Comment:
So you will automatically install pulseaudio if you have firefox installed, even if you don't want to use it? Or am I wrong here?